plugins/output/flight/flight.go

```diff
@@ -0,0 +1,156 @@
+package flight
+
+import (
+	"context"
+	_ "embed"
+	"log"
+	"net"
+
+	"github.com/apache/arrow/go/v9/arrow"
+	"github.com/apache/arrow/go/v9/arrow/array"
+	"github.com/apache/arrow/go/v9/arrow/flight"
+	"github.com/apache/arrow/go/v9/arrow/ipc"
+	"github.com/apache/arrow/go/v9/arrow/memory"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+// DO NOT REMOVE THE NEXT TWO LINES! This is required to embed the sampleConfig data.
+//
+//go:embed sample.conf
+var sampleConfig string
+
+func (*Flight) SampleConfig() string {
+	return sampleConfig
+}
+
+type Flight struct {
+	Location string `toml:"location"`
+	Port     string `toml:"port"`
+	Table    string `toml:"table"`
+
+	timeStamps []arrow.Timestamp
+	values     []float32
+
+	client flight.FlightServiceClient
+	writer flight.Writer
+	schema arrow.Schema
+	ctx    context.Context
+	pool   memory.GoAllocator
+	desc   *flight.FlightDescriptor
+}
+
+// Write should write immediately to the output, and not buffer writes
+// (Telegraf manages the buffer for you). Returning an error will fail this
+// batch of writes and the entire batch will be retried automatically.
+func (s *Flight) Write(metrics []telegraf.Metric) error {
```

Comment:
Maybe use more descriptive variable names than just single letters through the plugin.

Reply:
This is a method with a struct receiver and should always only be a single letter. 

I have yet to see it used any other way: 
[Example 1](https://gobyexample.com/methods)
[Example 2](https://github.com/influxdata/telegraf/blob/master/plugins/outputs/influxdb/influxdb.go)
[Example 3](https://www.golang-book.com/books/intro/9#section2)

Reply:
I have changed the letter to add clarity.

Reply:
Seem shorter [receiver](https://github.com/golang/go/wiki/CodeReviewComments#receiver-names) and [variable names](https://github.com/golang/go/wiki/CodeReviewComments#variable-names) are preferred when writing Go, so it is probably best to follow the current practice.